client/src/main/java/com/paypal/selion/testcomponents/AbstractPage.java

```diff
@@ -128,7 +128,7 @@ protected void loadObjectMap() {
                     objectMap = currentObjectMap;
                 }
             } catch (Exception e) {
-                throw new RuntimeException("Unable to initialize page data for " + pageDomain + "/" + pageClassName
+                throw new RuntimeException("Unable to initialize page data for " + pageDomain + "/" + pageClassName //NOSONAR
```

Comment:
The `//NOSONAR` in the middle of the `new RuntimeException` looks odd. Will you please declare a new variable and pass that to the call?